jgrapht-core/src/test/java/org/jgrapht/GraphTestsTest.java

```diff
@@ -555,34 +556,6 @@
         Graphs.addEdgeWithVertices(graph, 1, 3);
         assertTrue(GraphTests.isWeaklyChordal(graph));
     }
-
```

Comment:
Why are these tests being eliminated?

Reply:
They are not. My editor converted `@Test public void testIsWeaklyChordal()` to 
```
@Test
public void testIsWeaklyChordal()
```

Reply:
I'm referring to failRequireIsWeightedOnUnweightedGraph and failRequireIsWeightedOnNull

Reply:
I have no idea what happened. I've put the three tests back.